src/main/osd/osd.c

```diff
@@ -705,7 +705,7 @@ static bool osdDisplayStat(int statistic, uint8_t displayRow)
         else{
```

Comment:
😜 Found another :
`else {`
(in line 705)

Reply:
Let's wait for the final implementation‽ I like the idea very much!

Reply:
Thanks, I missed that one too.

Reply:
Done.

Reply:
😄